forge-gui/res/cardsfolder/k/kate_stewart.txt

```diff
@@ -2,7 +2,7 @@ Name:Kate Stewart
 ManaCost:1 U R W
 Types:Legendary Creature Human Scientist
 PT:3/3
-T:Mode$ CounterAddedOnce | ValidCard$ Permanent.YouCtrl | TriggerZones$ Battlefield | CounterType$ TIME | Execute$ TrigToken | TriggerDescription$ Whenever you put one or more time counters on a permanent you control, create a 1/1 white Soldier creature token.
+T:Mode$ CounterAddedOnce | ValidCard$ Permanent.YouCtrl+inZoneBattlefield | TriggerZones$ Battlefield | CounterType$ TIME | Execute$ TrigToken | TriggerDescription$ Whenever you put one or more time counters on a permanent you control, create a 1/1 white Soldier creature token.
```

Comment:
it's important you use inRealZone because otherwise there is some LKI logic so it would still trigger for something like _Epochrasite_